2018-11-03 19:38:47,487 ERROR: Exception on /api/settingDelete/ [POST] [in g:\termassignment\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "g:\termassignment\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "G:\TermAssignment\app\url_mapping.py", line 117, in settingDelete
    if DeleteArea(areaId):
  File "G:\TermAssignment\app\db_io.py", line 132, in DeleteArea
    db.session.delete(de)
  File "g:\termassignment\venv\lib\site-packages\sqlalchemy\orm\scoping.py", line 153, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "g:\termassignment\venv\lib\site-packages\sqlalchemy\orm\session.py", line 1810, in delete
    raise exc.UnmappedInstanceError(instance)
UnmappedInstanceError: Class '__builtin__.NoneType' is not mapped
2018-11-03 19:44:00,740 INFO: server startup [in G:\TermAssignment\app\__init__.py:45]
2018-11-03 19:44:13,816 ERROR: Exception on /api/settingNew/ [POST] [in g:\termassignment\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "g:\termassignment\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "G:\TermAssignment\app\url_mapping.py", line 105, in settingNew
    areaName = data['area']
KeyError: 'area'
2018-11-03 19:45:29,822 ERROR: Exception on /api/settingNew/ [POST] [in g:\termassignment\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "g:\termassignment\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "G:\TermAssignment\app\url_mapping.py", line 105, in settingNew
    areaName = data['area']
KeyError: 'area'
2018-11-03 19:48:05,385 ERROR: Exception on /api/settingDelete/ [POST] [in g:\termassignment\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "g:\termassignment\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "G:\TermAssignment\app\url_mapping.py", line 117, in settingDelete
    if DeleteArea(areaId):
  File "G:\TermAssignment\app\db_io.py", line 133, in DeleteArea
    db.session.delete(de)
  File "g:\termassignment\venv\lib\site-packages\sqlalchemy\orm\scoping.py", line 153, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "g:\termassignment\venv\lib\site-packages\sqlalchemy\orm\session.py", line 1810, in delete
    raise exc.UnmappedInstanceError(instance)
UnmappedInstanceError: Class '__builtin__.NoneType' is not mapped
2018-11-03 19:50:17,407 ERROR: Exception on /api/settingDelete/ [POST] [in g:\termassignment\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "g:\termassignment\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "G:\TermAssignment\app\url_mapping.py", line 117, in settingDelete
    if DeleteArea(areaId):
  File "G:\TermAssignment\app\db_io.py", line 133, in DeleteArea
    de = Area.query.filter_by(id=id).first()
  File "g:\termassignment\venv\lib\site-packages\sqlalchemy\orm\scoping.py", line 153, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "g:\termassignment\venv\lib\site-packages\sqlalchemy\orm\session.py", line 1810, in delete
    raise exc.UnmappedInstanceError(instance)
UnmappedInstanceError: Class '__builtin__.NoneType' is not mapped
2018-11-03 19:52:20,345 ERROR: Exception on /api/settingDelete/ [POST] [in g:\termassignment\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "g:\termassignment\venv\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "g:\termassignment\venv\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "G:\TermAssignment\app\url_mapping.py", line 117, in settingDelete
    if DeleteArea(areaId):
  File "G:\TermAssignment\app\db_io.py", line 133, in DeleteArea
    de = Area.query.filter_by(id=id).first()
  File "g:\termassignment\venv\lib\site-packages\sqlalchemy\orm\scoping.py", line 153, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "g:\termassignment\venv\lib\site-packages\sqlalchemy\orm\session.py", line 1810, in delete
    raise exc.UnmappedInstanceError(instance)
UnmappedInstanceError: Class '__builtin__.NoneType' is not mapped
2018-11-03 19:52:26,783 INFO: server startup [in G:\TermAssignment\app\__init__.py:45]
2018-11-03 19:56:52,030 INFO: server startup [in G:\TermAssignment\app\__init__.py:45]
